File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-jamba-development/JMBA_InComm_Android_SDK/app/src/main/java/com/wearehathway/apps/incomm/Models/InCommAutoReloadSavable.java
package com.wearehathway.apps.incomm.Models;

import java.util.Date;

/**
 * Created by vthink on 05/09/16.
 */
public class InCommAutoReloadSavable {
    private double Amount;
    private Date CreatedOn;
    private String EndsOn;
    private int GiftCardId;
    private int Id;
    private Boolean IsActive;
    private String LastErrorMessage;
    private int LastModifiedByPortalUserId;
    private Date LastModifiedBySystemOn;
    private Date LastModifiedByUserOn;
    private double MinimumBalance;
    private Date NextReloadOn;
    private int NumberOfOccurancesRemaining;
    private int PaymentAccountId;
    private String StartsOn;
    private String ReloadFrequencyId;


    public double getAmount() {
        return Amount;
    }

    public void setAmount(double amount) {
        Amount = amount;
    }

    public Date getCreatedOn() {
        return CreatedOn;
    }

    public void setCreatedOn(Date createdOn) {
        CreatedOn = createdOn;
    }



    public int getGiftCardId() {
        return GiftCardId;
    }

    public void setGiftCardId(int giftCardId) {
        GiftCardId = giftCardId;
    }

    public String getEndsOn() {
        return EndsOn;
    }

    public void setEndsOn(String endsOn) {
        EndsOn = endsOn;
    }

    public int getId() {
        return Id;
    }

    public void setId(int id) {
        Id = id;
    }

    public Boolean getActive() {
        return IsActive;
    }

    public void setActive(Boolean active) {
        IsActive = active;
    }

    public String getLastErrorMessage() {
        return LastErrorMessage;
    }

    public void setLastErrorMessage(String lastErrorMessage) {
        LastErrorMessage = lastErrorMessage;
    }

    public int getLastModifiedByPortalUserId() {
        return LastModifiedByPortalUserId;
    }

    public void setLastModifiedByPortalUserId(int lastModifiedByPortalUserId) {
        LastModifiedByPortalUserId = lastModifiedByPortalUserId;
    }

    public Date getLastModifiedBySystemOn() {
        return LastModifiedBySystemOn;
    }

    public void setLastModifiedBySystemOn(Date lastModifiedBySystemOn) {
        LastModifiedBySystemOn = lastModifiedBySystemOn;
    }

    public Date getLastModifiedByUserOn() {
        return LastModifiedByUserOn;
    }

    public void setLastModifiedByUserOn(Date lastModifiedByUserOn) {
        LastModifiedByUserOn = lastModifiedByUserOn;
    }

    public double getMinimumBalance() {
        return MinimumBalance;
    }

    public void setMinimumBalance(double minimumBalance) {
        MinimumBalance = minimumBalance;
    }

    public Date getNextReloadOn() {
        return NextReloadOn;
    }

    public void setNextReloadOn(Date nextReloadOn) {
        NextReloadOn = nextReloadOn;
    }

    public int getNumberOfOccurancesRemaining() {
        return NumberOfOccurancesRemaining;
    }

    public void setNumberOfOccurancesRemaining(int numberOfOccurancesRemaining) {
        NumberOfOccurancesRemaining = numberOfOccurancesRemaining;
    }

    public int getPaymentAccountId() {
        return PaymentAccountId;
    }

    public void setPaymentAccountId(int paymentAccountId) {
        PaymentAccountId = paymentAccountId;
    }

    public String getStartsOn() {
        return StartsOn;
    }

    public void setStartsOn(String startsOn) {
        StartsOn = startsOn;
    }

    public String getReloadFrequencyId() {
        return ReloadFrequencyId;
    }

    public void setReloadFrequencyId(String reloadFrequencyId) {
        ReloadFrequencyId = reloadFrequencyId;
    }
}

File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-Jamba-Android-v2.4.2/JMBA_InComm_iOS_SDK/InCommSDK/Models/Response/InCommAutoReloadSavable.swift
//
//  InCommAutoReloadSavable.swift
//  InCommSDK
//
//  Created by vThink on 8/10/16.
//  Copyright © 2016 Fishbowl. All rights reserved.
//

import Foundation
import SwiftyJSON

public struct InCommAutoReloadSavable{
    public var amount: Double
    public var createdOn: Date?
    public var endsOn: Date?
    public var giftCardId: Int32
    public var id: Int32
    public var isActive: Bool
    public var lastErrorMessage: String
    public var lastModifiedByPortalUserId: Int32
    public var lastModifiedBySystemOn: Date?
    public var lastModifiedByUserOn: Date?
    public var minimumBalance: Double?
    public var nextReloadOn: Double
    public var numberOfOccurancesRemaining: Int16?
    public var paymentAccountId: Int32
    public var startsOn: Date?
    public var reloadFrequencyId: InCommReloadFrequencyType
    
    public init(json:JSON){
        amount                      =  json["Amount"].doubleValue
        createdOn                   =  json["CreatedOn"].string?.dateFromInCommFormatString()
        endsOn                      =  json["EndsOn"].string?.dateFromInCommFormatString()
        giftCardId                  =  json["GiftCardId"].int32Value
        id                          =  json["Id"].int32Value
        isActive                    =  json["IsActive"].boolValue
        lastErrorMessage            =  json["LastErrorMessage"].stringValue
        lastModifiedByPortalUserId  =  json["LastModifiedByPortalUserId"].int32Value
        lastModifiedBySystemOn      =  json["LastModifiedBySystemOn"].string?.dateFromInCommFormatString()
        lastModifiedByUserOn        =  json["LastModifiedByUserOn"].string?.dateFromInCommFormatString()
        minimumBalance              =  json["MinimumBalance"].double
        nextReloadOn                =  json["NextReloadOn"].doubleValue
        numberOfOccurancesRemaining =  json["NumberOfOccurancesRemaining"].int16Value
        paymentAccountId            =  json["PaymentAccountId"].int32Value
        startsOn                    =  json["StartsOn"].string?.dateFromInCommFormatString()
        reloadFrequencyId           =  InCommReloadFrequencyType(rawValue: json["ReloadFrequencyId"].stringValue)!
    }
}
